benchmarks/versus-external-pools/fixed-worker-nodes.js

```diff
@@ -28,8 +28,7 @@ async function run () {
   // eslint-disable-next-line n/no-process-exit
   process.exit()
 }
-
-(async () => {
+;(async () => {
```

Comment:
There's a hard to reproduce bug in ts-standard that sometimes trigger that wrong change at post commit hook. Running pnpm lint:fix alone fix it.